setup.py

```diff
@@ -98,7 +97,7 @@
 
     # Installation
     # see https://www.python.org/dev/peps/pep-0345/#version-specifiers
-    python_requires=">=2.7,!=3.0,!=3.1,!=3.2,!=3.3",
+    python_requires=">=2.7,!=3.0,!=3.1,!=3.2,!=3.3,!=3.4",
```

Comment:
Does this mean it won't even try to install on Python 3.4? Do we want that?

Reply:
That's at least how we did it with Python 3.0-3.3.

> Does this mean it won't even try to install on Python 3.4?

And yes, apparently that's [how it should be handled](https://packaging.python.org/guides/distributing-packages-using-setuptools/#python-requires).

I guess we don't want that, since it might work and we just don't guarantee it. But then let's remove everything except `python_requires=">=2.7"`, right?

Reply:
I'm just surprised that no one complained about 3.0 through 3.3 not working! I'd have thought we'd get some issues if we remove support for `Python 3.4`... but then again people running Python3.4 probably aren't going to be running the latest release of python-can either.

I'm okay with you adding the `!=3.4`